data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Сэндс", "Sands" ],
+      [ "Пиментел", "Pimentel" ],
+      [ "Пиплз", "Peoples" ],
+      [ "Пирес", "Pires" ],
+      [ "Роуч", "Roach" ],
+      [ "Уир", "Weir" ],
+      [ "Карпентер", "Carpenter" ],
+      [ "Уэзерс", "Weathers" ],
+      [ "Ринг", "Ring" ],
+      [ "Филд", "Field" ],
+      [ "Баттс", "Butts" ],
+      [ "Портер", "Porter" ],
+      [ "Пратт", "Pratt" ],
+      [ "Принс", "Prince" ],
+      [ "Пэриш", "Parish" ],
+      [ "Форман", "Foreman" ],
+      [ "Рид", "Read" ],
+      [ "Пруэтт", "Pruett" ],
+      [ "Бёрд", "Bird" ],
+      [ "Пуарье", "Poirier" ],
+      [ "Крэйвен", "Craven" ],
+      [ "Бланк", "Blank" ],
+      [ "Пэрротт", "Parrott" ],
+      [ "Петти", "Petty" ],
+      [ "Бутчер", "Butcher" ],
+      [ "Ракер", "Rucker" ],
+      [ "Шелл", "Shell" ],
+      [ "Ралстон", "Ralston" ],
+      [ "Эрли", "Early" ],
+      [ "Ранхель", "Rangel" ],
+      [ "Раст", "Rust" ],
+      [ "Комбс", "Combs" ],
+      [ "Ратледж", "Rutledge" ],
+      [ "Редман", "Redman" ],
+      [ "Рентерия", "Renteria" ],
+      [ "Риверс", "Rivers" ],
+      [ "Рорк", "Roark" ],
+      [ "Робисон", "Robison" ],
+      [ "Хорн", "Horn" ],
+      [ "Роудс", "Rhodes" ],
+      [ "Рокуэлл", "Rockwell" ],
+      [ "Ролз", "Rawls" ],
+      [ "Ропер", "Roper" ],
+      [ "Роуленд", "Rowland" ],
+      [ "Роша", "Rocha" ],
+      [ "Гроув", "Grove" ],
+      [ "Гроувз", "Groves" ],
+      [ "Роу", "Roe" ],
+      [ "Роял", "Royal" ],
+      [ "Рапп", "Rupp" ],
+      [ "Фиш", "Fish" ],
+      [ "Рэглэнд", "Ragland" ],
+      [ "Рэгсдэйл", "Ragsdale" ],
+      [ "Рэйми", "Ramey" ],
+      [ "Рэнкин", "Rankin" ],
+      [ "Рэтлифф", "Ratliff" ],
+      [ "Саймонс", "Simons" ],
+      [ "Сальдивар", "Saldivar" ],
+      [ "Салмон", "Salmon" ],
+      [ "Самора", "Zamora" ],
+      [ "Сэмюэл", "Samuel" ],
+      [ "Бонд", "Bond" ],
+      [ "Сэлф", "Self" ],
+      [ "Сэвидж", "Savage" ],
+      [ "Норт", "North" ],
+      [ "Сент-Джон", "St. John" ],
+      [ "Сильвер", "Silver" ],
+      [ "Сильвестр", "Sylvester" ],
+      [ "Сильвия", "Silvia" ],
+      [ "Стронг", "Strong" ],
+      [ "Синглтэри", "Singletary" ],
+      [ "Стэйплс", "Staples" ],
+      [ "Слэк", "Slack" ],
+      [ "Слотэр", "Slaughter" ],
+      [ "Слейтер", "Slater" ],
+      [ "Хёрд", "Heard" ],
+      [ "Смоллвуд", "Smallwood" ],
+      [ "Соарес", "Soares" ],
+      [ "Спэйн", "Spain" ],
+      [ "Спрэйг", "Sprague" ],
+      [ "Старнс", "Starnes" ],
+      [ "Стейли", "Staley" ],
+      [ "Стейнберг", "Steinberg" ],
+      [ "Стэйтон", "Staton" ],
+      [ "Гласс", "Glass" ],
+      [ "Уолл", "Wall" ],
+      [ "Сент-Клер", "Stclair" ],
+      [ "Стовалл", "Stovall" ],
+      [ "Стовер", "Stover" ],
+      [ "Столлингс", "Stallings" ],
+      [ "Стоукс", "Stokes" ],
+      [ "Пейдж", "Page" ],
+      [ "Стрейндж", "Strange" ],
+      [ "Стрэттон", "Stratton" ],
+      [ "Стэнфилд", "Stanfield" ],
+      [ "Сунига", "Zuniga" ],
+      [ "Сэй", "Seay" ],
```

Comment:
According to [pronouncenames.com](https://www.pronouncenames.com/pronounce/seay), `Сэй` is just one of the ways to pronounce this surname. This may be due to it simply being pronounced differently in a regional or temporal sense.

Wikipedia provides diverse examples as well: [`Сиэй`](https://ru.wikipedia.org/wiki/%D0%A1%D0%B8%D1%8D%D0%B9,_%D0%94%D0%B6%D0%B5%D0%B9%D0%BC%D1%81) and [`Си`](https://ru.wikipedia.org/wiki/%D0%A1%D0%B8,_%D0%90%D0%BB%D1%8C%D0%B1%D0%B5%D1%80%D1%82).

I genuinely have no idea what to go with, here. I suppose `Сэй` is fine.